.tekton/pull-request.yaml

```diff
@@ -259,6 +260,8 @@ spec:
                 # Perform cleanup of resources created by gitops service
                 oc delete --ignore-not-found deployment --all -n $(params.e2e_test_namespace)
                 oc delete --ignore-not-found eventlisteners --all -n $(params.e2e_test_namespace)
+  timeouts:
+    pipeline: "1h30m"
```

Comment:
same for here as well, to fully utilize the e2e-tests task timeout + time taken by other tasks.. increase the timeout here as well?

Reply:
thanks! updated.